lib/taurus/qt/qtgui/input/tauruslineedit.py

```diff
@@ -201,6 +201,29 @@ def _stepBy(self, v):
         value = self.getValue()
         self.setValue(value + Quantity(v, value.units))
 
+    def getDisplayValue(self, cache=True, fragmentName=None):
```

Comment:
This is not critical , but... most taurus code avoids using super because of some issues with diamond inheritance schemes, so I would prefer the following implementation:

```python
def getDisplayValue(self, cache=True, fragmentName=None):
    if fragmentName is None and self.modelFragmentName is None:
        fragmentName = 'wvalue'
    return TaurusBaseWidget.getDisplayValue(self, cache=cache, fragmentName=fragmentName)
```

Reply:
Then this TaurusValueLineEdit.getDisplayValue() may call the newer TaurusBaseWritableWidget.getDisplayValue() instead of the supersuperclass. Even it's an empty call right now, if this is propagated up one would realise better the scale.

Reply:
100% right. It should be TaurusBaseWritableWidget.getDisplayValue

My mistake